url.bs

```diff
@@ -457,8 +457,9 @@ up to three <a>ASCII digits</a> per sequence, each representing a decimal number
 
      XXX should we define the format inline instead just like STD 66? -->
 
-<p>A <dfn export>valid opaque-host string</dfn> must be one or more <a>URL units</a> or: U+005B ([),
-followed by a <a>valid IPv6-address string</a>, followed by U+005D (]).
+<p>A <dfn export>valid opaque-host string</dfn> must be one or more <a>URL units</a>, excluding
```

Comment:
I think this would be clearer as a bulleted list (one bullet for each branch of the or)